### ISSUE TITLE
Discussion: secure_scan id numbering

### ISSUE THREAD

**Issue by agvandervelde** [Paradigm4/revealcore#9]
Hi! Not strictly revealcore-related, but putting it out here. If we have set permissions on certain IDs for certain users, there might be a situation where we reload (part) of the protected data. We should probably ensure that when we do that, we load secure scan IDs such that they do not collide with existing, invalidated access record, lest we end up exposing the wrong data to the wrong person.

**Comment by mpietras-p4** [comment]
Yeah, that's something that the appropriate `delete` functions will need to handle - e.g., in revealgenomics https://github.com/Paradigm4/revealgenomics/blob/master/R/delete.R#L547
